docs/faq/ImmutableData.md

```diff
@@ -464,9 +463,7 @@ Operating on JavaScript objects and arrays in an immutable way can be slow, part
 
 Remember, to change an immutable object, you must mutate a _copy_ of it, and copying large objects can be slow as every property must be copied.
 
-In contrast, immutable libraries such as Immutable.JS can employ sophisticated optimization techniques such as [structural sharing](https://www.slideshare.net/mohitthatte/a-deep-dive-into-clojures-data-structures-euroclojure-2015) , which effectively returns a new object that reuses much of the existing object being copied from.
-
-For copying very large objects, [plain JavaScript can be over 100 times slower](https://medium.com/@dtinth/immutable-js-persistent-data-structures-and-structural-sharing-6d163fbd73d2#.z1g1ofrsi) than an optimized immutable library.
+In contrast, immutable libraries such as Immer can employ sophisticated optimization techniques such as [structural sharing](https://www.slideshare.net/mohitthatte/a-deep-dive-into-clojures-data-structures-euroclojure-2015) , which effectively returns a new object that reuses much of the existing object being copied from.
```

Comment:
:hand: Immer doesn't use any "sophisticated data structures". It _does_ say it uses "structural sharing", but Immer uses that phrase in a different way than the Immutable.js docs do.

We should probably drop the link to that "Clojure data structures" article, and I don't think the phrase "sophisticated optimization techniques" applies here.  Really, all it's doing is copying only the parts of the object structure that need to be changed, and reusing the ones that aren't changing.